share/picongpu/examples/FoilLCT/include/picongpu/param/particle.param

```diff
@@ -32,8 +32,7 @@
 #include "picongpu/particles/manipulators/manipulators.def"
 #include "picongpu/particles/traits/GetAtomicNumbers.hpp"
 
-#include <pmacc/nvidia/functors/Add.hpp>
-#include <pmacc/nvidia/functors/Assign.hpp>
+#include <pmacc/math/operation.hpp>
```

Comment:
I think this is unused and can be removed.

Reply:
same as before, I would keep it to allow easy copy-pasting